docs/environment-setup.md

```diff
@@ -168,6 +168,15 @@ _Note: If experiencing an error such as `Error starting userland proxy: listen t
 
 To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
 
-### Finally
+### Finally:
 
+Telescope requires a running back-end and front-end to start.
+Start the backend for Telescope:
 Run `npm start` and open `localhost:3000`
+
+Depending on how you wish to run the front-end, different methods are available. For more information please refer to to our [Front-End Document](gatsbyjs.md).
+If the `npm run develop` command was used:
+open `localhost:8000`
+
+If the `npm run build` command was used:
+Run `npm run serve` and open `localhost:9000`
```

Comment:
This only starts the server, without processing feeds. Also, if the user previously ran `npm start`, this will throw since `start` and `server` will both try to serve on port `3000` (where's that `9000` defined?).

Reply:
Really? I was able to run both command with no errors

![image](https://user-images.githubusercontent.com/18711727/96169818-b7970600-0ef0-11eb-8ada-63046d2002d7.png)

Reply:
I think you're using `serve` from the front-end's `package.json`, while the documentation refers to `server` in the back-end's.
In any case, back-end's `npm start` will both process feeds and serve the front-end, so no need for an extra terminal running anything else (when using `npm run build`).